gensrc/proto/cloud.proto

```diff
@@ -662,6 +663,7 @@ message CommitTxnResponse {
     repeated int64 partition_ids = 4;
     repeated int64 versions = 5;
     repeated TableStatsPB table_stats = 6;
+    optional int64 version_update_time = 7;
```

Comment:
What is this for? Why is version_update_time related to RecycleTxnPB

Reply:
They are for batch and single respectively. What makes you think the relation ever existed?